src/ziggurat/streams.clj

```diff
@@ -31,6 +31,10 @@
    :oldest-processed-message-in-s      604800
    :changelog-topic-replication-factor 3})
 
+(def KEY_DESERIALIZER_ENCODING "key.deserializer.encoding")
+(def VALUE_DESERIALIZER_ENCODING "value.deserializer.encoding")
+(def DESERIALIZER_ENCODING "deserializer.encoding")
```

Comment:
instead of def, define these using let in the function required

Reply:
This is done.